pkg/apis/acme/v1/types_issuer.go

```diff
@@ -118,9 +118,12 @@ type ACMEExternalAccountBinding struct {
 	// encoded data.
 	Key cmmeta.SecretKeySelector `json:"keySecretRef"`
 
-	// keyAlgorithm is the MAC key algorithm that the key is used for.
-	// Valid values are "HS256", "HS384" and "HS512".
-	KeyAlgorithm HMACKeyAlgorithm `json:"keyAlgorithm"`
+	// (deprecated) keyAlgorithm is the MAC key algorithm that the key is used
+	// for. Valid values are "HS256", "HS384" and "HS512".
+	// Deprecation warning: this value is no longer used as
```

Comment:
some tools look for `Deprecated:` in comments: https://github.com/golang/go/wiki/Deprecated

it might be worth changing `Deprecation warning:` to `Deprecated:` for this reason - there are several [examples](https://github.com/golang/go/search?q=%22Deprecated%3A%22) of this in the standard library in golang/go

Reply:
Thanks! I guess those descriptions are mostly used as descriptions of the CRD types, but agree that it makes sense to standardize. I've changed the deprecation warnings on all 3 api versions + the internal type.